mongosql/src/air/desugarer/testdata/desugar_accumulators.yml

```diff
@@ -59,6 +59,111 @@ tests:
           },
       }
 
+  - name: "desugar non-distinct $sqlCount single column arg must be doc"
+    input:
+      - {
+        "$group":
+          {
+            "_id": null,
+            "acc": { "$sqlCount": { "var": "$a", "distinct": false, "arg_is_possibly_doc": "must" } },
+          },
+      }
+    expected:
+      - {
+        "$group":
+          {
+            "_id": null,
+            "acc":
+              {
+                "$sum":
+                  {
+                    "$cond":
+                      [
+                        {
+                          "$or":
```

Comment:
I flipped this from a conjunction of negative conditions to a disjunction of positive ones. This just helped line up the single- and multiple-column cases more easily, especially now that we account for single-columns that may be documents.